ui/src/app/pages/login/account-creation/account-creation.spec.tsx

```diff
@@ -1,22 +1,39 @@
 import * as React from 'react';
 import * as fp from 'lodash/fp';
-import { mount, ReactWrapper } from 'enzyme';
+import { mount, ReactWrapper, shallow, ShallowWrapper } from 'enzyme';
 
 import { createEmptyProfile } from 'app/pages/login/sign-in';
 import colors from 'app/styles/colors';
 import { serverConfigStore } from 'app/utils/stores';
 
-import { AccountCreation, AccountCreationProps } from './account-creation';
+import {
+  AccountCreation,
+  AccountCreationProps,
+  countryDropdownOption,
+  stateCodeErrorMessage,
+} from './account-creation';
 
 let props: AccountCreationProps;
 const component = () => {
   return mount(<AccountCreation {...props} />);
 };
 
+const shallowComponent = () => {
```

Comment:
would be a good idea to add comment why we are using shallow and not mount

Reply:
Interesting, I was wondering the exact opposite! Do we usually default to `mount`? From what I understand, `mount` typically runs slower and is only used when you need functionality not implemented in `shallow`, such as some lifecycle methods.

From the enzymejs docs:

"[Shallow rendering](https://enzymejs.github.io/enzyme/docs/api/shallow.html) is useful to constrain yourself to testing a component as a unit, and to ensure that your tests aren't indirectly asserting on behavior of child components"

"[Full DOM rendering](https://enzymejs.github.io/enzyme/docs/api/mount.html) is ideal for use cases where you have components that may interact with DOM APIs or need to test components that are wrapped in higher order components."

Reply:
Thanks for explaining  this. I realized we are using both, going to check if we are using mount/shallow in correctly or not